lib/ui/top_app_bar.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import '../main.dart';
+
+class TopBar extends StatefulWidget implements PreferredSizeWidget {
+  final String title;
+
+  const TopBar(
+      {Key? key, required this.title}): super(key:key);
+
+  Size get preferredSize => Size.fromHeight(50.0);
+
+  //const TopBar({required String this.title});
```

Comment:
We should probably remove this, since it's dead/commented code.

Reply:
Sounds good, thanks for the suggestion.